packages/trader/src/AppV2/Utils/layout-utils.tsx

```diff
@@ -50,3 +50,12 @@ export const getChartHeight = ({
         return height - HEIGHT.ADDITIONAL_INFO;
     return height;
 };
+
+export const removeFocus = (
+    e?: React.MouseEvent<HTMLInputElement, MouseEvent>,
+    ref?: React.RefObject<HTMLInputElement>
+) => {
+    (e?.target as HTMLElement)?.blur();
+    ref?.current?.blur();
+    (document?.activeElement as HTMLElement)?.blur();
```

Comment:
You can remove focus by passing event or ref ot in case if nothing was passed it'll still remove focus, but not from specific element.